app_legacy/Helpers/database/set-claim.php

```diff
@@ -358,6 +356,7 @@ function getFilteredClaims(
     $selectCondition = "
         sc.ID AS ID,
         sc.User AS User,
+        ua.Permissions AS UserRole,
```

Comment:
What is the need for exposing `ua.Permissions`? That probably won't translate easily into the new permissions matrix logic. Looking at the spreadsheet being replaced/augmented, it seems like it would be sufficient to have a `bool UserIsJrDev` property.

Reply:
I understand your concern. Apologies for my weakness with SQL. Is something like this in the query what you're describing?:

```sql
CASE WHEN ua.Permissions = 2 THEN true ELSE false END AS UserIsJrDev
```

Reply:
Yes.

I'm not sure if it would be better to use "<= 2" instead of "= 2", as users with 0 or 1 Permissions shouldn't have claims, but if they did, they should be treated as Jr. Devs.

Reply:
I have added the above clause to the query in https://github.com/RetroAchievements/RAWeb/pull/1435/commits/343f3c8b1241c7e1c2d17e9c517e620168f10885.

I went with `<= 2`, since it does indeed seem possible a user with less than `2` can have a claim if they are demoted or banned.

Reply:
You're still exposing `UserRole`, which may not translate well once we move to a permission matrix.

Reply:
Oversight on my part, forgot to delete the line 🤦‍♂️
Should be resolved now.